packages/nameguard-react/src/index.ts

```diff
@@ -11,3 +11,5 @@ export { NameShield } from "./components/NameShield";
 export { Shield } from "./components/Shield/Shield";
 export { Shield as ShieldIcon } from "./components/Report/Shield";
 export { LoadingIcon as LoadingShield } from "./components/icons/Loading";
+
+import "@namehash/ens-webfont";
```

Comment:
Will this cause any conflicts in NameKit?

Reply:
I can't think of any possibility of conflicts